src/components/results/loadflow/limit-violation-result.tsx

```diff
@@ -286,18 +300,12 @@ export const LimiViolationResult: FunctionComponent<LoadflowResultProps> = ({
         );
     };
 
-    const voltageViolations = overloadedEquipments.filter(
-        (overloadedEquipment) =>
-            overloadedEquipment.limitType === LimitTypes.HIGH_VOLTAGE ||
-            overloadedEquipment.limitType === LimitTypes.LOW_VOLTAGE
-    );
-
     useEffect(() => {
         //To avoid the rapid flashing of "no rows" before we actually show the rows
-        if (voltageViolations && isFetchComplete) {
+        if (overloadedEquipments && isFetchComplete) {
```

Comment:
By using the same state to check if the equipments are ready for both current and voltage violation in two useEffects, you lost the purpose of these useEffects and still have this weird double rendre of the table when we change the tab